cli/src/semgrep/scan_report.py

```diff
@@ -116,7 +116,7 @@ def _print_scan_plan_header(
             evolve(target_manager, baseline_handler=None).get_all_files()
         )
         diff_file_count = len(target_mode_config.get_diff_targets())
-        summary_line = f"Pro Differential Scanning {diff_file_count}/{unit_str(total_file_count, 'file')}"
+        summary_line = f"Pro Differential Scanning {unit_str(diff_file_count, 'file')}"
```

Comment:
Why changing the message format here?

Reply:
I think showing only the number of changed files is less confusing, so I simplified it.

Reply:
Perhaps it should be "Inter-file Differential ..." instead of "Pro Differential ..." ?

Reply:
done.